unicorecmsmnmcameroon/templates/base.pt

```diff
@@ -18,27 +17,19 @@
 </head>
 <body>
     <div id="header">
-        <div class="lang" tal:condition="len(view.get_display_languages())>1">
-          <span tal:repeat="item view.get_display_languages()">
-            <a tal:attributes="{'class':'active'} if view.locale==item[0] else {}" href="/locale/${item[0]}/">${item[1]}</a> |
-          </span>
-          <a href="/locale/change/" tal:condition="len(view.get_available_languages)>2" class="lang-change">&hellip;</a>
-        </div>
-        <div metal:use-macro="view.logo_template" tal:omit-tag="" tal:define="img_attrs view.get_logo_attributes(default_image_src=request.static_url('cms:static/img/ConnectSmart_UNICEF-logo-footer.png'))">
+        <div metal:use-macro="view.logo_template" tal:omit-tag="" tal:define="img_attrs view.get_logo_attributes(default_image_src=request.static_url('cms:static/img/mnm-cameroon-logo.png'))">
         </div>
         <div class="clr"></div>
     </div>
     <div class="push"></div><!--/PUSH-->
-    <div id="submenu" metal:define-slot="nav">
-        <a href="/" i18n:translate="Home">Home</a>
-        <span tal:repeat="cat view.get_top_nav">
-            | <a href="/content/list/${cat.uuid}/">${cat.title}</a>
-        </span>
-    </div>
 
     <div id="content">
         <div metal:define-slot="content"></div>
     </div>
-    <div id="base"><img alt='' src="/static/img/ConnectSmart_UNICEF-logo-footer.png" /></div><!--/BASE-->
+    <div id="base">
+            Copyright &copy;2014<br/>
+            Malaria No More<br/>
+            <a href="/contact/" i18n:translate="Contact">Contact Us</a>
```

Comment:
The translate string should be the same as the string i.e `i18n:translate="Contact Us"`